libs/judicial-system/types/src/lib/case.ts

```diff
@@ -14,6 +14,7 @@ export enum CaseOrigin {
 export enum CaseType {
   CUSTODY = 'CUSTODY',
   TRAVEL_BAN = 'TRAVEL_BAN',
+  ADMISSION_TO_FACILITY = 'ADMISSION_TO_FACILITY',
```

Comment:
The const caseTypes in formatters.ts in the formatters lib needs to be updated as well.